docs/source/deploying.rst

```diff
@@ -84,59 +84,113 @@ The following resources explain how to set up Dask on a variety of local and dis
+
+-   `Coiled (recommended) <https://coiled.io?utm_source=dask-docs&utm_medium=deploying>`_:
+    this commercial SaaS product handles most of the deployment pain we currently
+    see, is easy to use, and quite robust.  The free tier is generous enough
+    for most individual users, even for those who don't want to engage with a
+    commercial company.  The API looks like the following.
+
+    .. code-block:: python
+
+       import coiled
+       cluster = coiled.Cluster(
+           n_workers=100,
+           region="us-east-2",
+           worker_memory="16 GiB",
+           spot_policy="spot_with_fallback",
+       )
+       client = cluster.get_client()
+
+- `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_: a pure and simple OSS solution that sets up Dask workers on cloud VMs, supporting AWS, GCP, Azure, and also other commercial clouds like Hetzner and Digital Ocean.
+
+- `Dask-Yarn <https://yarn.dask.org>`_: deploys Dask on legacy YARN clusters, such as can be set up with AWS EMR or Google Cloud Dataproc
 
 See :doc:`deploying-cloud` for more details.
 
 .. _Coiled: https://coiled.io?utm_source=dask-docs&utm_medium=deploying
-.. |Coiled| replace:: **Coiled** 
+.. |Coiled| replace:: **Coiled**
 
 
 High Performance Computing
 --------------------------
-|Dask-Jobqueue|_ **is recommended for deploying Dask on HPC systems.**
-Though there are other options you may consider depending on your specific needs:
 
-- `Dask-Jobqueue <https://jobqueue.dask.org>`_: Provides cluster managers for PBS, SLURM, LSF, SGE and other resource managers.
-- `Dask-MPI <http://mpi.dask.org/en/latest/>`_: Deploy Dask from within an existing MPI environment.
+Dask runs on traditional HPC systems that use a resource manager like SLURM,
+PBS, SGE, LSF, or similar systems, and a network file system.  This is an easy
+way to dual-purpose large-scale hardware for analytics use cases.  Dask can
+deploy either directly through the resource manager or through
+``mpirun``/``mpiexec`` and tends to use the NFS to distribute data and
+software.
+
+-   `Dask-Jobqueue (recommended) <https://jobqueue.dask.org>`_: interfaces directly with the
+    resource manager (SLURM, PBS, SGE, LSF, and others) to launch many Dask
+    workers as batch jobs.  It generates batch job scripts and submits them
+    automatically to the user's queue.  This approach operates entirely with user
+    permissions (no IT support required) and enables interactive and adaptive use
+    on large HPC systems.  It looks a little like the following:
+
+    .. code-block:: python
+
+       from dask_jobqueue import PBSCluster
+
+       cluster = PBSCluster(
+           cores=24,
+           memory="100GB",
+           queue="regular",
+           account="my-account",
+       )
+       cluster.scale(jobs=100)
+
+       client = cluster.get_client()
```

Comment:
[Nit] Removing extra spaces to match the other example snippets (or add extra spaces to the other example) 

```suggestion
       from dask_jobqueue import PBSCluster
       cluster = PBSCluster(
           cores=24,
           memory="100GB",
           queue="regular",
           account="my-account",
       )
       cluster.scale(jobs=100)
       client = cluster.get_client()
```